slidez-fe-core/src/common/components/forms/LoginForm.tsx

```diff
@@ -55,9 +66,9 @@ const LoginForm = ({ onLogin, onLoginWithGoogle }: LoginProps) => {
                     <Form>
                         <div className='form-row form-input-holder'>
                             <div
-                                className={loginError ? 'error-text' : 'hidden'}
+                                className={loginError ? '' : 'hidden'}
                             >
-                                {"Can't log in: email or password is invalid"}
+                            {() => loginFailed(errors.email)}
```

Comment:
Check it after login request, here it will be shown on any page update if login is failed